events/wc_undead_events.txt

```diff
@@ -1598,12 +1851,35 @@ character_event = {
 		name = EVTOPTA_WCUND_375
 
 		remove_racial_traits_effect = yes
-		add_trait = creature_abomination
+		
+		# Assign creature trait depending on religion		
+		if = {
+			limit = { 
+				NOT = {
+					religion = throsic
+				}
+			}
+			add_trait = creature_abomination
+		}
+		
+		if = {
+			limit = { religion = throsic }
+			add_trait = creature_wicker_golem
+		}
 
 		# Fixes immortal age
 		add_age_back_effect = yes
 
+		# Set graphical culture depending on creature type
+		if = {
+			trait = creature_abomination
+		}
 		hidden_effect = { set_graphical_culture = abomination }
+		
+		if = {
+			trait = creature_wicker_golem
+		}
+		hidden_effect = { set_graphical_culture = wicker_golem }
```

Comment:
The syntax isn't correct at all, so it doesn't work.
https://ck2.paradoxwikis.com/Scripting#If